sql/hive/src/test/scala/org/apache/spark/sql/hive/CachedTableSuite.scala

```diff
@@ -193,6 +193,54 @@ class CachedTableSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
     }
   }
 
+  test("Refresh Qualified Tables") {
+    withTempDatabase { db =>
+      withTempView("cachedTable") {
+        try {
+          sql(s"CREATE TABLE $db.cachedTable STORED AS PARQUET AS SELECT 1")
+          sql(s"CACHE TABLE $db.cachedTable")
+          assertCached(sql(s"select * from $db.cachedTable"), s"`$db`.`cachedTable`")
+          assert(spark.catalog.isCached(s"$db.cachedTable"), s"Table '$db.cachedTable' should be cached")
+
+          sql(s"REFRESH TABLE $db.cachedTable")
+          assertCached(sql(s"select * from $db.cachedTable"), s"`$db`.`cachedTable`")
+          assert(spark.catalog.isCached(s"$db.cachedTable"), s"Table '$db.cachedTable' should be cached after refresh")
+
+          activateDatabase(db) {
+            assertCached(sql("select * from cachedTable"), s"`$db`.`cachedTable`")
+            assert(spark.catalog.isCached("cachedTable"), "Table 'cachedTable' should be cached after refresh")
+
+            sql(s"REFRESH TABLE cachedTable")
+            assertCached(sql("select * from cachedTable"), s"`$db`.`cachedTable`")
+            assert(spark.catalog.isCached("cachedTable"), "Table 'cachedTable' should be cached after refresh")
+          }
+        } finally {
+          sql(s"UNCACHE TABLE $db.cachedTable")
+          sql(s"DROP TABLE $db.cachedTable")
+        }
+
```

Comment:
Nit: delete empty line

Reply:
Thanks. The file is updated and the empty line is removed.